lib/CreateStudent.dart

```diff
@@ -1,135 +1,1160 @@
+                                    borderSide: BorderSide(
+                                        color: Colors.black, width: 1.0),
+                                  ),
+                                  enabledBorder: OutlineInputBorder(
+                                    borderSide: BorderSide(
+                                        color: Colors.black, width: 1.0),
+                                  ),
+                                  hintText: 'Phone Number-2',
+                                  labelText: "Phone Number-2"),
+                              validator: (val) {
+                                if (val == null || val.isEmpty) {
+                                  return 'Please enter registration number';
+                                } else if (!validPhoneNo) {
+                                  return 'Please enter valid Phone. No.';
+                                }
+                              },
+                            )),
+                        Padding(
+                            padding: EdgeInsets.only(
+                                bottom: 6.0, left: 60, right: 60, top: 10),
+                            child: TextFormField(
+                              decoration: new InputDecoration(
+                                  isDense: true,
+                                  contentPadding: new EdgeInsets.symmetric(
+                                      vertical: 10.0, horizontal: 10.0),
+                                  focusedBorder: OutlineInputBorder(
+                                    borderSide: BorderSide(
+                                        color: Colors.black, width: 1.0),
+                                  ),
+                                  enabledBorder: OutlineInputBorder(
+                                    borderSide: BorderSide(
+                                        color: Colors.black, width: 1.0),
+                                  ),
+                                  hintText: 'E-Mail Address',
+                                  labelText: "E-mail."),
+                              validator: (val) {
+                                if (val == null || val.isEmpty) {
+                                  return 'Please enter Email';
+                                }
+                              },
+                              onChanged: (value) => {
+                                setState(() {
+                                  if (EmailValidator.validate(value)) {
+                                    validEmail = true;
+                                  } else {
+                                    validEmail = false;
+                                  }
+                                }),
+                              },
+                            )),
+                        Padding(
+                            padding: EdgeInsets.only(
+                                bottom: 6.0, left: 60, right: 60, top: 10),
```

Comment:
same changes required for student page as well

Reply:
ok